### ISSUE TITLE
Error in gtk-update-icon-cache with PKGBUILD (0.2.7.8) in Arch

### ISSUE THREAD

**Issue by fjospinas** [OrancheloTeam/oranchelo-icon-theme#9]
I excetute makepkg and pacman -U and I obtain the next error.

```
(1/1) instalando oranchelo-icon-theme-git          [######################] 100%
:: Ejecutando los «hooks» de posinstalación...
(1/1) Updating icon theme caches...
gtk-update-icon-cache: The generated cache was invalid.
error: la orden no se ejecutó correctamente
```

Then I look folder by folder /usr/share/icons con:

cd /usr/share/icons
for f in $(ls */index.theme | sed 's,/index.theme$,,' ) ; do  echo $f ;  gtk-update-icon-cache -f  $f || echo XXX $f XXX ; echo ; done

And I obtain the same error in oranchelo folder:

```
Adwaita
gtk-update-icon-cache: Cache file created successfully.

default

gnome
gtk-update-icon-cache: Cache file created successfully.

hicolor
gtk-update-icon-cache: Cache file created successfully.

HighContrast
gtk-update-icon-cache: Cache file created successfully.

Maia-Cursor
gtk-update-icon-cache: Cache file created successfully.

Oranchelo
gtk-update-icon-cache: The generated cache was invalid.
XXX Oranchelo XXX

Simple-and-Soft

Vanilla-DMZ-AA

Vertex-Maia
gtk-update-icon-cache: Cache file created successfully.

```

Finally the package is installed and it works but I want to remove the error.

**Comment by fjospinas** [comment]
I tried to create the oranchelo-icon-theme-git.install file to create the cache manually post installation scripts but does not work. 

The file content is:

```
post_install() {
  gtk-update-icon-cache
}

post_upgrade() {
  post_install
}

post_remove() {
  post_install
}
```

**Comment by Madh93** [comment]
The problem is related about some icon names with spaces.

![](http://oi63.tinypic.com/xbz1uv.jpg)

It's necessary fixed it.

**Comment by unhammer** [comment]
these are the space-ones in the ppa:
```
$ dpkg -L oranchelo-icon-theme|grep ' '
/usr/share/icons/Oranchelo/apps/scalable/gtk tehemes.svg
/usr/share/icons/Oranchelo/apps/scalable/liri-browser 2.svg
/usr/share/icons/Oranchelo/status/gtk-dialog-warning (copia).svg
/usr/share/icons/Oranchelo/status/dialog-warning-symbolic (copia).svg
/usr/share/icons/Oranchelo/status/dialog-warning (copia).svg
/usr/share/icons/Oranchelo/apps/scalable/Nylas N1.svg
/usr/share/icons/Oranchelo/apps/scalable/unity-control-center screen.svg
/usr/share/icons/Oranchelo/apps/scalable/nylas n1.svg
/usr/share/icons/Oranchelo/apps/scalable/nylas N1.svg
/usr/share/icons/Oranchelo/apps/scalable/Angry Birds-chrome.angrybirds.com.svg
```

**Comment by Madh93** [comment]
Removed from source code, this should be fixed in the next release. Thanks for report :+1:

**Comment by Madh93** [comment]
Solved in [v0.7.3.5](https://github.com/OrancheloTeam/oranchelo-icon-theme/releases/tag/v0.7.3.5).
